### PR TITLE
update lists of members

### PR DESCRIPTION
**Pull request by Ori6575**

### CODE REVIEW

File: module.lua [hunk 1]
```diff
@@ -61,7 +61,7 @@ local _TEAM = {
 		["Sebaisseba#0020"] = "EN",
 		["Shamousey#0015"] = "EN",
 		["Sharpiepoops#0020"] = "EN",
-		["Tat#0020"] = "CN",
+		["Tat#0095"] = "CN",
```
Comment (Ori6575) [inline module.lua]: ex staff

File: module.lua [hunk 2]
```diff
@@ -73,7 +73,7 @@ local _TEAM = {
 	fc = {
 		["Acer#0010"] = "EN",
 		["Adami#0010"] = "BR",
-		["Aeroman#0095"] = "EN",
+		["Aeroman#0010"] = "EN",
```
Comment (Ori6575) [inline module.lua]: rejoined the mod team

File: module.lua [hunk 3]
```diff
@@ -109,43 +109,40 @@ local _TEAM = {
 		["Eyeground#0000"] = "FR",
 		["Faz_x#0010"] = "PL",
 		["Flare#0010"] = "NL",
-		["Flashpoint#0015"] = "EN",
```
Comment (Ori6575) [inline module.lua:112]: Accounts Switch:
Flashpoint#0015 => Censere#0015

File: module.lua [hunk 4]
```diff
@@ -109,43 +109,40 @@ local _TEAM = {
...
+		["Karbi#0095"] = "ES",
 		["Katburger#0015"] = "EN",
 		["Kathryn#8850"] = "EN",
 		["Katow#0020"] = "FR",
 		["Kim#0010"] = "EN",
 		["Kiwrimai#0015"] = "BR",
 		["Kloure#0010"] = "EN",
 		["Kurt#0010"] = "BR",
-		["Lament#0015"] = "PL",
-		["Lapinprince#0015"] = "EN",
```
Comment (Ori6575) [inline module.lua:143]: Idk if he needs to be here, dont know if he's still fc on this account.

File: module.lua [hunk 5]
```diff
@@ -245,30 +243,27 @@ local _TEAM = {
 		["Coska#0015"] = "PL",
 		["Doraemons#0010"] = "CN",
 		["Dracoleaf#0010"] = "EN",
+		["Eagles0ng#0010"] = "RU",
 		["Etyla#0015"] = "FR",
 		["Excasr#0010"] = "BR",
-		["Flashpoint#0015"] = "EN",
```
Comment (Ori6575) [inline module.lua]: Accounts Switch:
Flashpoint -> Censere

File: module.lua [hunk 6]
```diff
@@ -245,30 +243,27 @@ local _TEAM = {
...
 		["Gavin#0015"] = "EN",
 		["Grastfetry#0015"] = "BR",
 		["Hollya#0015"] = "FR",
 		["Hotaru#0015"] = "RO",
-		["Jacob#0010"] = "EN",
-		["Jerry#0015"] = "RU",
+		["Inkzooka#0015"] = "EN",
 		["Jordy#0010"] = "NL",
 		["Katburger#0015"] = "EN",
-		["Kcalb#0015"] = "BR",
```
Comment (Ori6575) [inline module.lua:259]: Trial account of Sollyun, trial ended.

File: module.lua (hunk 4 above)
Comment (Lautenschlager-id) [reply module.lua:143]: Yes, they are. Add Lapinprince#0015 back.

@edit: I'll add it myself

**Review by Lautenschlager-id — APPROVED** [review]

**Pull request merged**